ceph-multibranch-pipeline/config/definitions/ceph-multibranch-pipeline.yml

```diff
@@ -0,0 +1,16 @@
+---
+- job:
+    name: ceph-multibranch-pipeline
+    project-type: multibranch
+    number-to-keep: 600
+    days-to-keep: 30
+    scm:
+      - github:
+          repo: ceph
+          repo-owner: ceph
+          credentials-id: 8cffdeb4-283c-4d96-a190-05d5645bcc2f
```

Comment:
is it OK to have this here?

Reply:
Yes.  It's just a UUID Jenkins uses to identify which encrypted credentials to use.

Reply:
I think this is the 'name'/id of the credentials. Perhaps @djgalloway knows a better way of putting this?

Reply:
> I think this is the 'name'/id of the credentials. Perhaps @djgalloway knows a better way of putting this?

It's fine :+1: